packages/commons/src/config/httpServiceConfig.ts

```diff
@@ -27,8 +27,10 @@ export const HTTPServerConfig = z
   .object({
     HOST: APIEndpoint,
     PORT: z.coerce.number().min(1001),
+    KEEP_ALIVE_TIMEOUT_MILLIS: z.coerce.number().default(5),
```

Comment:
```suggestion
    KEEP_ALIVE_TIMEOUT_MILLIS: z.coerce.number().default(5000),
```
5 seconds (https://nodejs.org/api/http.html#serverkeepalivetimeout)